### PR TITLE
VEGEMITE

### PR DESCRIPTION
**Pull request by Pacmandevil**
VEGEMITE

### CODE REVIEW

**Comment by Pacmandevil** [comment]
fucking reeee

**Comment by Pacmandevil** [comment]
I don't know how I fucked up this badly

**Comment by Pacmandevil** [comment]
i can't beleive u done this

**Comment by Lohikar** [comment]
```
 _    ______________________  ___________________
| |  / / ____/ ____/ ____/  |/  /  _/_  __/ ____/
| | / / __/ / / __/ __/ / /|_/ // /  / / / __/
| |/ / /___/ /_/ / /___/ /  / // /  / / / /___
|___/_____/\____/_____/_/  /_/___/ /_/ /_____/

```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 1]
```diff
@@ -157,6 +157,21 @@
 	..()
 	M.intoxication += removed*0.02 //Very slightly alcoholic
 
+/datum/reagent/nutriment/coating/vegemite
+	name = "Vegemite"
+	cooked_name = "burnt Vegemite"
+	id = "Vegemite"
```
Comment (Lohikar) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: Maybe lowercase this.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 2]
```diff
@@ -157,6 +157,21 @@
...
+	color = "#0c0901"
+	taste_description = "Concentrated industrial grade salt"
```
Comment (Lohikar) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: Should probably be `concentrated` (lowercase).

**Comment by KepQwet** [comment]
```

 _    _ _______  ______ _______ _______ _____ _______ _______
  \  /  |______ |  ____ |______ |  |  |   |      |    |______
   \/   |______ |_____| |______ |  |  | __|__    |    |______
                                                             


```

**Comment by Pacmandevil** [comment]
reeeee

**Comment by Pacmandevil** [comment]
Closing due to my brain damage

**Pull request closed without merging**